charms/argo-controller/src/charm.py

```diff
@@ -80,10 +80,7 @@ def main(self, event):
         self.model.unit.status = MaintenanceStatus("Setting pod spec")
 
         # Sync the argoproj/argoexec image to the same version
-        metadata = yaml.safe_load(Path("metadata.yaml").read_bytes())
-        version = metadata["resources"]["oci-image"]["upstream-source"].split(":")[-1]
-        executor_image = f"argoproj/argoexec:{version}"
-        self.log.info(f"using executorImage {executor_image}")
```

Comment:
Should we still keep this log message?

Reply:
I'd say that we don't since one can see it doing `juju config argo-controller`.

Reply:
Yeah, I thought about it and that would work.